dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -44,6 +44,34 @@ rocommunity {{ snmp_rocommunity }}
 rocommunity6 {{ snmp_rocommunity }}
 {% endif %}
 
+{% if SNMP_COMMUNITY is defined %}
```

Comment:
if there is same community string is defined in yaml file and config_db, will there be a problem?
What will be the behavior if the type is different in yaml and config_db?

Reply:
Good feedback. I've set this up to update the snmp.yml file and configdb but the jinja template will only get the communities from the snmp.yml file until we can cutover.  This way there is no issues.

Reply:
Updated the Jinja file to only include info from the configdb.  
With the start.sh file running the snmp_yml_to_configdb.py we get all the info from the snmp.yml file.
This way there is no conflict.  I've also set the snmp_yml_to_configdb.py to prefer the configdb configs over the snmp.yml file so if there is a change and the configdb is save it will prefer the configdb.